src/Components/StopWatch/StopWatch.tsx

```diff
@@ -4,8 +4,8 @@ import ControlButtons from "../ControlButton";
 import Timer from "../Timer";
   
 function StopWatch() {
-  const [isActived, setIsActived] = useState(false);
-  const [isPaused, setIsPaused] = useState(true);
+  const [isActived, setIsActived] = useState(false); // For Check Press Reset Button
```

Comment:
Since the variable name is `isActive`, it actually means whether the timer is active or not.
There's a saying "Code itself is a comment"